shared/lib_shared_inverter.cpp

```diff
@@ -211,66 +210,68 @@ void SharedInverter::calculateACPower(const double powerDC_kW_in, const double D
-	for (size_t i = 0; i < powerDC_kW_in.size(); i++)
-		powerDC_Watts_one_inv.push_back(powerDC_kW_in[i] * util::kilowatt_to_watt/ m_numInverters);
+    //need to convert to watts and divide power by m_num_inverters
+    std::vector<double> powerDC_Watts_one_inv;
+    for (size_t i = 0; i < powerDC_kW_in.size(); i++)
+        powerDC_Watts_one_inv.push_back(powerDC_kW_in[i] * util::kilowatt_to_watt / m_numInverters);
 
-	// Power quantities go in and come out in units of W
-	double powerAC_Watts = 0;
-	if (m_inverterType == SANDIA_INVERTER || m_inverterType == DATASHEET_INVERTER || m_inverterType == COEFFICIENT_GENERATOR)
-		m_sandiaInverter->acpower(powerDC_Watts_one_inv, DCStringVoltage, &powerAC_Watts, &P_par, &P_lr, &efficiencyAC, &powerClipLoss_kW, &powerConsumptionLoss_kW, &powerNightLoss_kW);
-	else if (m_inverterType == PARTLOAD_INVERTER)
-		m_partloadInverter->acpower(powerDC_Watts_one_inv, &powerAC_Watts, &P_lr, &P_par, &efficiencyAC, &powerClipLoss_kW, &powerNightLoss_kW);
+    // Power quantities go in and come out in units of W
+    double powerAC_Watts = 0;
+    if (m_inverterType == SANDIA_INVERTER || m_inverterType == DATASHEET_INVERTER || m_inverterType == COEFFICIENT_GENERATOR)
+        m_sandiaInverter->acpower(powerDC_Watts_one_inv, DCStringVoltage, &powerAC_Watts, &P_par, &P_lr, &efficiencyAC, &powerClipLoss_kW, &powerConsumptionLoss_kW, &powerNightLoss_kW);
+    else if (m_inverterType == PARTLOAD_INVERTER)
+        m_partloadInverter->acpower(powerDC_Watts_one_inv, &powerAC_Watts, &P_lr, &P_par, &efficiencyAC, &powerClipLoss_kW, &powerNightLoss_kW);
 
-	Tdry_C = tempC;
+    Tdry_C = tempC;
     StringV = DCStringVoltage[0];
     double tempLoss = 0.0;
-	if (m_tempEnabled){
-		//use average of the DC voltages to pick which temp curve to use- a weighted average might be better but we don't have that information here
-		double avgDCVoltage = 0;
-		for (size_t i = 0; i < DCStringVoltage.size(); i++)
-			avgDCVoltage += DCStringVoltage[i];
-		avgDCVoltage /= DCStringVoltage.size();
-		calculateTempDerate(avgDCVoltage, tempC, powerAC_Watts, efficiencyAC, tempLoss);
-	}
-
-	// Scale to total system size
-	// Do not need to scale back up by m_numInverters because scaling them down was a separate vector, powerDC_Watts_one_inv
-	powerDC_kW = 0;
-	for (size_t i = 0; i < powerDC_kW_in.size(); i++)
-		powerDC_kW += powerDC_kW_in[i];
-
-	//Convert units to kW and scale to total array for all other outputs
-	convertOutputsToKWandScale(tempLoss, powerAC_Watts);
+    double power_ratio = 1.0;
+    if (m_tempEnabled) {
+        //use average of the DC voltages to pick which temp curve to use- a weighted average might be better but we don't have that information here
+        double avgDCVoltage = 0;
+        for (size_t i = 0; i < DCStringVoltage.size(); i++)
+            avgDCVoltage += DCStringVoltage[i];
+        avgDCVoltage /= DCStringVoltage.size();
+        calculateTempDerate(avgDCVoltage, tempC, powerAC_Watts, power_ratio, tempLoss);
```

Comment:
tempLoss will now be in DC. So any comments and the var_info table label should specify that